src/components/layout/HeaderBanner/HeaderBanner.jsx

```diff
@@ -13,27 +14,53 @@ import LicenseEdition from './LicenseEdition';
  *
  * HeaderBanner is a component displaying application header.
  *
- * * contains: logo, product name, license edition and version
- * * supports theming: `headerTextColor` parameter is used as text color
+ * * contains: logo, product name, license edition and product version
+ * * supports theming:
+ *   * `headerTextColor` parameter is used as text color
+ *   * `logoUrl` parameter is used as URL for Logo component
+ *   * `showVersionDetails` parameter is used to determine if license edition and product version should be visible
  */
-export default function HeaderBanner({ licenseEdition, logoUrl, productName, productVersion, showVersionDetails }) {
+export default function HeaderBanner({
+    className,
+    licenseEdition,
+    logoUrl,
+    productName,
+    productVersion,
+    showVersionDetails
+}) {
     const theme = useContext(ThemeContext);
 
+    let color = colors.white;
+    if (theme && theme.headerTextColor) {
+        color = theme.headerTextColor;
+    }
+
+    let url = logoUrl;
+    if (theme && theme.logoUrl) {
+        url = theme.logoUrl;
+    }
+
+    let showDetails = showVersionDetails;
+    if (theme && _.isBoolean(theme.showVersionDetails)) {
+        showDetails = theme.showVersionDetails;
+    }
```

Comment:
I'd use `{}` as default value for `theme` and then:
```suggestion
    const color = theme.headerTextColor || colors.white;
    const url = theme.logoUrl || logoUrl;
    const showDetails = _.isBoolean(theme.showVersionDetails) ?  theme.showVersionDetails : showVersionDetails;
```

Reply:
Makes sense. Fixed.